packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -292,9 +292,9 @@ class AndroidGradleBuilder implements AndroidBuilder {
 
     final bool usesAndroidX = isAppUsingAndroidX(project.android.hostAppGradleRoot);
     if (usesAndroidX) {
-      BuildEvent('app-using-android-x', flutterUsage: _usage).send();
+      BuildEvent('app-using-android-x', type: 'gradle', flutterUsage: _usage).send();
```

Comment:
This should probably be close to 99%. @xster, maybe it's not very useful now?

Reply:
In the last month it was 99.24%